apps/ledger-live-desktop/src/renderer/screens/account/index.tsx

```diff
@@ -220,7 +221,7 @@ const AccountPage = ({
             )
           ) : null}
           {isOrdinalsEnabled && account.type === "Account" && account.currency.id === "bitcoin" ? (
-            <OrdinalsAccount account={account} />
+            <OrdinalsAccount account={account as BitcoinAccount} />
```

Comment:
Why force?

Reply:
Because I need to have the bitcoinRessources that are only inside BitcoinAccount and I thought it would be cleaner to do it this way to avoid ts error. I can add a utils to check if it's a btc account by checking if the account has `bitcoinResources`

Reply:
Changed for two type guards => one that checks if it's a fork of btc and one to check if that fork is a BTC account or a fork